src/uwtools/resources/jsonschema/esg-grid.jsonschema

```diff
@@ -0,0 +1,119 @@
+{
+  "allOf": [
+    {
+      "if": {
+        "not": {
+          "required": [
+            "namelist.base_file"
+          ]
+        }
+      },
```

Comment:
Added in an IF - THEN conditional to confirm that when the `base_file` is not supplied, we will require all the `namelist` options be required in the `uodates_values` object.